_posts/2020-05-26-exploring-the-open-sgid-a-brief-tutorial-using-open-source-python-tools.md

```diff
@@ -0,0 +1,146 @@
+---
+title: 'Exploring the Open SGID: A Brief Tutorial Using Open Source Python Tools'
+author:
+  display_name: Erik Neemann
+  email: eneemann@utah.gov
+date: 2020-05-26 16:17:43
+categories:
+  - SGID Blog
+tags:
+  - open sgid
+  - python
+  - geopandas
+  - open source
+  - notebook
+  - tutorial
+---
+
+The accompanying Python Notebook for this blog post can be found [on Google Colab](https://colab.research.google.com/drive/13evGMipBC0lqFStboTkZmtJji-zvovs0#scrollTo=tEkj3tFP-ca2).
+
+### The Open SGID
+
+The introduction of the [Open SGID](https://gis.utah.gov/introducing-open-sgid/) has opened the door for new ways to explore Utah's GIS data.  In the past, a user would typically have gone to the AGRC web page to find the data they were looking for and download it in shapefile or file geodatabase format--and this can still be done today.  Now that AGRC has leveraged the [Open Data](https://opendata.gis.utah.gov/) platform, there are more options for users to access geospatial data in their preferred way.  Data can now be streamed into a GIS client via web services or downloaded in even more formats (GeoJSON, KML, CSV).  But another exciting offering is the recently launched Open SGID, which is a cloud-hosted PostgreSQL PostGIS database that anyone can connect to.  With nightly automated updates, tapping into the Open SGID means you'll have a living connection to the freshest SGID data available.
+The Open SGID also creates an opportunity to pull in data and perform analysis using only open source Python libraries.  With this route, you don't need to worry about paying for licensed software or even installing GIS desktop software at all.  And if you use a cloud-hosted Jupyter notebook--like Google Colab--you don't even have to install Python on your computer (or tablet or chromebook)!
+
+### Python Notebook and Libraries
+
+The rest of this blog will walk you through a few examples of how easy it can be to pull SGID data into Python to perform analysis or create simple maps.  These basic building blocks, combined with a little gumption and googling, can put you well on your way to performing whatever analysis you desire.  A link to the Jupyter notebook in Google Colab is also included, so you can run the code yourself and build upon it.  Following Python libraries will be used in this tutorial:
```

Comment:
```suggestion
The rest of this blog will walk you through a few examples of how easy it can be to pull SGID data into Python to perform analysis or create simple maps.  These basic building blocks, combined with a little gumption and googling, can put you well on your way to performing whatever analysis you desire.  A link to the Jupyter notebook in Google Colab is also included, so you can run the code yourself and build upon it.  The following Python libraries will be used in this tutorial:
```